spec/helpers/session.rb

```diff
@@ -0,0 +1,20 @@
+module SessionHelpers
+
+  def sign_up(user)
+    visit '/users/new'
+    expect(page.status_code).to eq(200)
+    fill_in :email, with: user.email
+    fill_in :handle, with: user.handle
+    fill_in :password, with: user.password
+    fill_in :password_confirmation, with: user.password_confirmation
+    click_button 'Sign up'
+  end
+
+  def sign_in(handle:, password:)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `handle`. You can also write as `sign_in(*)` if you want the method to accept any arguments but don't care about them.<br>Lint/UnusedMethodArgument: Unused method argument - `password`. You can also write as `sign_in(*)` if you want the method to accept any arguments but don't care about them.